.github/workflows/autolabel_action.yaml

```
@@ -0,0 +1,39 @@
+name: Apply at most one `t-`label to PRs
+
+on: pull_request
+
+jobs:
+  apply_one_t_label:
+    runs-on: ubuntu-latest
+    # Don't run on forks, where we wouldn't have permissions to add the label anyway.
+    if: github.repository == 'leanprover-community/mathlib4'
+    permissions:
+      issues: write
+      checks: write
+      pull-requests: write
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - run: |
+          ./scripts/autolabels.sh long
+          printf 'Just the labels: "%s"\n' "$(./scripts/autolabels.sh)"
```

Let's make this easier to understand for first-time readers:
```suggestion
          printf 'About to add the following labels: "%s"\n' "$(./scripts/autolabels.sh)"